scripts/run-backend-tests

```diff
@@ -88,11 +88,16 @@ cd backend && \
   "${PRODEXEC}" migrations run > "$LOGS/test-migrations.log" 2>&1
 cd ..
 
-DARK_CONFIG_TELEMETRY_EXPORTER=none \
-DARK_CONFIG_ROLLBAR_ENABLED=n \
-DARK_CONFIG_QUEUE_PUBSUB_PROJECT_ID=pubsub-test-${RANDOM_VALUE} \
-DARK_CONFIG_TRACE_STORAGE_BUCKET_NAME=trace-test-${RANDOM_VALUE} \
-./scripts/build/reload-packages --test $PUBLISHED_FLAG
+# Reload packages
+
+if [[ "$PUBLISHED" == "true" ]]; then
```

Comment:
Is there a way this can be 'if running in CI' rather than 'if running the published assets'?
Sometimes, though rarely, we'll run published assets locally, which I think will cause this to fail